sdk/communication/Azure.Communication.Administration/src/PhoneNumberSearchPurchaseOperation.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Communication.Administration.Models;
+
+namespace Azure.Communication.Administration
+{
+    /// <summary>
+    /// Represent a long-running phone number search purchase operation.
+    /// </summary>
+    public class PhoneNumberSearchPurchaseOperation : Operation<PhoneNumberSearch>
```

Comment:
Let's rename to `PurchaseReservationOperation` and not return the `PhoneNumberSearch` model as discussed offline.